MetaGenerator/UserInput/ExistingPathRequester.cs

```diff
@@ -14,9 +15,7 @@ public ExistingPathRequester(IValidatedUserInput validatedUserInput)
         }
 
         protected override IEnumerable<IValidation> RequesterValidations(IUserInputDescription variable)
-        {
-            yield return new SimpleValidation(PathExists, "The path you have given does not exists, please enter an existing path:");
-        }
+            => Sequence.Return(SimpleValidation.Create(PathExists, "The path you have given does not exists, please enter an existing path:"));
```

Comment:
What is the colon doing at the end?

Reply:
Having a party with his characterly friends!